src/devices/GamepadMappings.js

```diff
@@ -65,9 +65,9 @@ let oculusGo = {
 let oculusTouch = {
   mapping: 'xr-standard',
   displayProfiles: {
-    'Oculus Quest': ['oculus-quest', 'oculus-touch', 'thumbstick-controller']
+    'Oculus Quest': ['oculus-quest', 'oculus-touch', 'generic-trigger-squeeze-thumbstick']
```

Comment:
You'll have to check with Artem, but i"m pretty sure he didn't ever want to report 'oculus-quest' as a profileId

Reply:
`oculus-quest` is still in the registry, which is what I was referencing. I can change it if that's the confirmed preference, though.